### PR TITLE
[Dygraph] split yolov3 loss and update yolov3

### PR DESCRIPTION
**Pull request by wangxinxin08**
1. split yolov3 loss
2. add sync_bn and modify list to LayerList to use sync_bn
3. support cutmix op in dataset
4. add missing op
5. modify reader to use new op

### CODE REVIEW

File: ppdet/data/transform/operator.py [hunk 1]
```diff
@@ -182,6 +182,55 @@ def apply(self, sample, context=None):
...
+@register_op
+class RandomErasingImageOp(BaseOperator):
+    def __init__(self, prob=0.5, sl=0.02, sh=0.4, r1=0.3):
+        """
+        Random Erasing Data Augmentation, see https://arxiv.org/abs/1708.04896
+        Args:
+            prob (float): probability to carry out random erasing
+            sl (float): lower limit of the erasing area ratio
+            sh (float): upper limit of the erasing area ratio
+            r1 (float): aspect ratio of the erasing region
```
Comment (heavengate) [inline ppdet/data/transform/operator.py:194]: 这个参数会呈现的配置文件里面，感觉可以换个自注释点的命名

File: ppdet/data/source/dataset.py [hunk 2]
```diff
@@ -39,13 +39,29 @@ def __init__(self,
         self.image_dir = image_dir if image_dir is not None else ''
         self.sample_num = sample_num
         self.use_default_label = use_default_label
+        self.kwargs = kwargs
```
Comment (heavengate) [inline ppdet/data/source/dataset.py:42]: 用`kwargs`的话感觉直接在`__init__`里把`mixup_epoch`解出来比较好，不需要把`kwargs`维护在成员变量里面

File: ppdet/data/reader.py [hunk 3]
```diff
@@ -101,18 +101,22 @@ def __call__(self,
                  dataset,
                  worker_num,
                  device,
+                 sampler=None,
```
Comment (heavengate) [inline ppdet/data/reader.py:104]: `sampler` -> `batch_sampler`

File: ppdet/modeling/__init__.py [hunk 4]
```diff
@@ -19,3 +20,4 @@
 from .architecture import *
 from .post_process import *
 from .layers import *
+from .utils import *
```
Comment (qingqing01) [inline ppdet/modeling/__init__.py:23]: modeling里没有 utils https://github.com/PaddlePaddle/PaddleDetection/tree/dygraph/ppdet/modeling

File: ppdet/data/transform/operator.py (hunk 1 above)
Comment (wangxinxin08) [reply ppdet/data/transform/operator.py:194]: Done, thanks!

File: ppdet/data/source/dataset.py (hunk 2 above)
Comment (wangxinxin08) [reply ppdet/data/source/dataset.py:42]: Done, thanks!

File: ppdet/data/reader.py (hunk 3 above)
Comment (wangxinxin08) [reply ppdet/data/reader.py:104]: Done, Thanks!

**Pull request closed without merging**